src/cli/console/helpers.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+
+from src.cli import console
+from src.graphql import GraphQL
+
+
+def organization_id_2_display_name(ctx, id: str = None) -> str:
```

Comment:
@buschNT This triggers an api request every single time? Is this necessary? I think we should persist/cache these titles locally and only call the api if there is no value stored locally.
If you keep the caching/retrieveing logic in these functions, we can leave the rest (calling code) as it is.